monai/handlers/transform_inverter.py

```diff
@@ -114,42 +112,17 @@ def __call__(self, engine: Engine) -> None:
         Args:
             engine: Ignite Engine, it can be a trainer, validator or evaluator.
         """
-        for output_key, batch_key, nearest_interp, to_tensor, device, post_func in zip(
-            self.output_keys, self.batch_keys, self.nearest_interp, self.to_tensor, self.device, self.post_func
-        ):
-            transform_key = batch_key + InverseKeys.KEY_SUFFIX
-            if transform_key not in engine.state.batch:
-                warnings.warn(f"all the transforms on `{batch_key}` are not InvertibleTransform.")
-                continue
-
-            transform_info = engine.state.batch[transform_key]
-            if nearest_interp:
-                transform_info = convert_inverse_interp_mode(
-                    trans_info=deepcopy(transform_info),
-                    mode="nearest",
-                    align_corners=None,
-                )
-
-            output = engine.state.output[output_key]
-            if isinstance(output, torch.Tensor):
-                output = output.detach()
-            segs_dict = {
-                batch_key: output,
-                transform_key: transform_info,
-            }
-            meta_dict_key = f"{batch_key}_{self.meta_key_postfix}"
-            if meta_dict_key in engine.state.batch:
-                segs_dict[meta_dict_key] = engine.state.batch[meta_dict_key]
-
-            with allow_missing_keys_mode(self.transform):  # type: ignore
-                inverted = self.inverter(segs_dict)
+        # combine `batch` and `output` to temporarily act as 1 dict for post transform
+        data = dict(engine.state.batch)
```

Comment:
I think, we have to update the docs saying that the engine should have `state.batch` and `state.output` as dicts, otherwise it will fail here for any engine (e.g. those where batch is a tuple of 2 items).

Reply:
Hi @vfdev-5 ,

Thanks for your review!
Yes, you made a good point, I added more doc-string in the latest commit.

Thanks.